packages/manager/apps/dedicated/package.json

```
@@ -25,6 +25,7 @@
     "@ovh-ux/manager-at-internet-configuration": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-banner": "^1.1.3",
     "@ovh-ux/manager-beta-preference": "^1.0.0",
+    "@ovh-ux/manager-billing": "^0.0.0-alpha || ^1.0.0",
```

Shouldn't it be something like 
```suggestion
    "@ovh-ux/manager-billing": "^0.0.0 || ^0.1.0-alpha",
```
?